doc/extending/graphstructures.txt

```diff
@@ -399,7 +402,7 @@ In both types of pairs, the second element of the tuple is an index,
 such that: ``var.clients[*][0].inputs[index]`` or
 ``fgraph.outputs[index]`` is that variable.
 
-.. code-block:: python
+.. testcode::
```

Comment:
The comments below like:

```
# [(Sum(Elemwise{add,no_inplace}.0), 0)]
```
Are actually things that would be outputs were that entered at the python prompt.

Maybe the section should be converted to a `>>>` block.